react/more_react_concepts/refs_and_memoization.md

```diff
@@ -62,7 +62,7 @@ useEffect(() => {
 
 The interesting thing about this is that this will **never** trigger a component re-render!
 
-Another question that might pop up in your mind is, "Why not just use `querySelector` or other DOM manipulation methods that we've done previously in vanilla JavaScript?" Dealing with the DOM ourselves defeats the purpose of using React, and wherever possible we should let React _commit_ to the DOM itself.
+Another question that might pop up in your mind is, "Why not just use `querySelector` or other DOM manipulation methods that we've done previously in vanilla JavaScript?" Dealing with the DOM ourselves defeats the purpose of using React, and wherever possible we should let React *commit* to the DOM itself.
```

Comment:
is this done by a linter? I do see this syntax change in more reviews and it doesn't look like there's anything about cursive/italic in our style guide so this should be fine. 

It's generally best not to let a linter go through files I believe to avoid mismatching style guides :)